CONTRIBUTING.md

```diff
@@ -47,7 +47,7 @@ There's also other Video.js projects where you can help. (check the [video.js or
 
 [GitHub Issues](https://github.com/videojs/video.js/issues) are used for all discussions around the codebase, including **bugs**, **features**, and other **enhancements**.
 
-When filling out an issue, make sure to fill out the questions in the
+When filling out an issue, please include as much information as possible.
```

Comment:
```suggestion
When filling out an issue, please respond to all of the questions in the template, including as much information as possible.
```

Thanks! Even small changes like this are useful and welcome. I've made a suggestion, which you can accept just by clicking "commit suggestion".

By the way, you'll find it more manageable to make your changes in a new [branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches) in your fork rather than on your `main` branch.